torch/onnx/symbolic_caffe2.py

```diff
@@ -1,20 +1,21 @@
 import importlib
 from inspect import getmembers, isfunction
 
-import torch.onnx.symbolic_helper as sym_help
-import torch.onnx.symbolic_registry as sym_registry
+from torch.onnx import symbolic_helper
+from torch.onnx import symbolic_opset9 as opset9
```

Comment:
nit: why is symbolic_opset9 treated as special case abbreviating away `symbolic`?

Reply:
I found the module used very often in ops, and `opset9` seems to be a good balance between the naming principles and aesthetics/readability to me. To be precise, it's also not an abbreviation, since `symbolic_opset9` is two words. It is acceptable to take words out, or give them an alias if it is an "inconveniently long name". Just don't take letters out from a word. (I just took the liberty to decide symbolic_opset9 is "inconveniently long.") Happy to do it the principle way (use the module name) though.

What do you think?

Reply:
sgtm